File: Language_Coder/src/function/Code578.java
package function;

import java.util.Scanner;

public class Code578 {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		int num1, num2;
		num1 = scan.nextInt();
		num2 = scan.nextInt();

		if (num1 >= num2) {
			int temp = num1;
			num1 = num2;
			num2 = temp;
		}

		for (int i = num1; i <= num2; i++) {
			System.out.println("== " + i + "dan ==");
			for (int j = 1; j <= 9; j++) {
				System.out.print(i + " * " + j + " =");
				if (i * j < 10)
					System.out.println("  " + i * j);
				else
					System.out.println(" " + i * j);
			}
		}
	}

}

File: Language_Coder_Cpp/Code578/Code578.cpp
// Code578.cpp 

#include <iostream>
using namespace std;

void multiply(int num) {
	cout << "== " << num << "dan ==" << endl;
	for (int i = 1; i <= 9; i++) {
		cout << num << " * " << i << " =";
		if (num * i >= 10) {
			cout << " ";
		}
		else {
			cout << "  ";
		}
		cout << num * i << endl;
	}
	cout << endl;
}

int main()
{
	int num1, num2;
	cin >> num1 >> num2;

	if (num1 >= num2) {
		int temp = num1;
		num1 = num2;
		num2 = temp;
	} // swap

	for (int i = num1; i <= num2; i++) {
		multiply(i);
	}
}